woof-code/rootfs-skeleton/usr/share/doc/HOWTO-multimedia.htm

```diff
@@ -6,6 +6,8 @@
 	<link href="./manual.css" rel="stylesheet" type="text/css">
 </head>
 <body>
+	<a href="#" class="scroll_back_button" target="_self">&uArr;</a>
```

Comment:
Make Pmusic-specific documentation optional, should be inside woof-code/rootfs-packages/pmusic

Reply:
I have currently not updated this file. I will open up another PR for the file.